src/encoded/commands/create_annotated_filenames.py

```diff
@@ -654,11 +650,29 @@ def get_aliquot_id_from_samples(tissue_samples: List[Dict[str], Any]) -> Filenam
     aliquot_ids = [
         get_aliquot_id_from_tissue_sample(sample) for sample in tissue_samples
     ]
+    if len(aliquot_ids) > 1:
+        aliquot_ids = get_multiple_aliquot_id_from_samples(aliquot_ids)
     return get_filename_part_for_values(
         aliquot_ids, "tissue aliquot ID", source_name="sample"
     )
 
 
+def get_multiple_aliquot_id_from_samples(ids: List[str]):
```

Comment:
As I am not clear on how the decision on this ended up and the desired result I leave it up to you to make sure that this is doing as expected.